lightkurve/lightcurve.py

```diff
@@ -1085,3 +1086,153 @@ def opt_depth(to, width, data):
         snr_d.append(depth_star * np.sqrt(width_star))
 
     return log_posterior, trial_periods, trial_periods[np.argmax(log_posterior)]
+
+class LightCurveCollection(object):
+    """
+    Collects multiple LightCurve objects together with helpful functions.
+    """
+
+    def __init__(self, lcs):
+        try:
+            lightcurves = np.asarray(lcs)
+        except:
+            raise TypeError("Unable to parse input")
+        self.data = {}
+        for lc in lightcurves:
+            try:
+                if lc.keplerid:
+                    self.data[lc.keplerid] = lc
+            except:
+                raise TypeError("Object is not a LightCurve instance")
+
+    def __len__(self):
+        return len(self.data)
+
+    def _ids(self):
+        """
+        Returns the kepler_ids of all the lightcurves as a dict_keys obj.
+        """
+        return self.data.keys()
+
+    def __getitem__(self, kep_id):
+        """
+        Returns the lightcurve associated with the kepler_id. 
+        """
+        try: 
+            return self.data[kep_id]
+        except:
+            raise ValueError('No LightCurve for ' + kep_id)
+
+    def append(self, lc):
+        try:
+            self.data[lc.keplerid] = lc
+        except:
+            raise TypeError("Input is not a lightcurve")
+
+    def __repr__(self):
+        result = ""
+        for lightcurve in self.data.values():
+            result += lightcurve.__repr__() + "\n"
+        return result
+
+    def plot(self):
```

Comment:
Let's have plot take in an optional `ax` and `**kwargs` so it behaves like `lc.plot()`